cmd/rhobs/requests.go

```
@@ -0,0 +1,697 @@
+
+func (p *csvLogsPrinter) PrintHeader() {
+	header := []string{}
+	if p.isPrintingTimeValue {
+		header = append(header, "TIME")
+	}
+	for _, fieldName := range p.fieldNames {
+		header = append(header, fieldName)
+	}
+	header = append(header, "MESSAGE")
+
+	p.writer.Write(header)
```

golangci-lint (2.11.4)

[error] Error return value of `p.writer.Write` is not checked

(errcheck)